### PR TITLE
Clarify a few things around the spec

### PR DESCRIPTION
**Pull request by brian-brazil**
Based on various feedback over the last year clarify some wording, and where spec and code deviated.
See commits for more details.

### CODE REVIEW

File: specification/OpenMetrics.md [hunk 1]
```diff
@@ -440,6 +441,9 @@ Line feed, '\n' (0x0A) -> literally '\n' (Bytecode 0x5c 0x6e)
 Double quotes -> '\"' (Bytecode 0x5c 0x22)
 Backslash -> '\\' (Bytecode 0x5c 0x5c)
 
+You SHOULD NOT use a backslash before other characters, as they do not escape.
+'\\a' SHOULD be used instead of the equivialent '\a' for example.
+
```
Comment (RichiH) [inline specification/OpenMetrics.md:446]:
```suggestion
Double backslashes SHOULD be used to escape characters. Single backslash SHOULD NOT be used, but will work for well-defined escape sequences.
As an example, '\\a' escapes and is preferable to '\a'. As another example, '\\n' is preferable even though '\n' also escapes.

```

**Review by RichiH — APPROVED** [review]
LGTM other than suggested change.

File: specification/OpenMetrics.md (hunk 1 above)
Comment (brian-brazil) [reply specification/OpenMetrics.md]: Your suggestion doesn't make sense as-is, but I've taken the thrust of it.

File: specification/OpenMetrics.md [hunk 2]
```diff
@@ -440,6 +441,10 @@ Line feed, '\n' (0x0A) -> literally '\n' (Bytecode 0x5c 0x6e)
 Double quotes -> '\"' (Bytecode 0x5c 0x22)
 Backslash -> '\\' (Bytecode 0x5c 0x5c)
 
+A double backslashe SHOULD be used to represent a backslash character.
+Single backslash SHOULD NOT be used for undefined escape sequences, but will work.
+As an example, '\\a' is equivilent and preferable to '\a'.
```
Comment (RichiH) [inline specification/OpenMetrics.md:446]:
```suggestion
A double backslash SHOULD be used to represent a backslash character.
A single backslash SHOULD NOT be used for undefined escape sequences.
As an example, '\\a' is equivalent and preferable to '\a'.
```

Comment (brian-brazil) [reply specification/OpenMetrics.md:446]: Fixed, and also spotted escaping issues in rendered version which I've fixed too.

**Review by RichiH — APPROVED** [review]

**Pull request merged**